### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by NurkaAmre**
### Desktop version
- You need use @media query
- Setup size 1024px
- When you apply media query please check mobile version is not spoiled.
- Images for desktop version use background-images.
- Follow the style as Figma.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -316,3 +304,310 @@ footer {
...
+    margin-right: 0;
+    flex: 1;
+  }
+
+  .hero-section1 {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    margin-bottom: 100px;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css:367]: - [ ] Great job with your design 👍 . Kindly make sure your hero text is vertically centered as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 2]
```diff
@@ -316,3 +304,310 @@ footer {
...
+
+  .project-item {
+    position: relative;
+  }
+
+  .container-information {
+    position: absolute;
+    width: 100%;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css:441]: - [ ] Kindly make sure you use the correct font size and weigh in the projects section 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 3]
```diff
@@ -316,3 +304,310 @@ footer {
...
+    margin-top: 0;
+  }
+
+  .last-form {
+    display: grid;
+    grid-template-columns: 1fr 0.01fr 1fr;
+  }
+
+  .last-grid1 {
+    padding-right: 70px;
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Kindly make sure you use the appropriate margin and spacing as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @NurkaAmre, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — COMMENTED** [review]
# Hi @NurkaAmre  😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

## **Invalid Submission**

- Please fix the Linters errors

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

**Comment by bellom** [comment]
> ### Desktop version
> * You need use @media query
> * Setup size 1024px
> * Be patience

It would be nice to have a more descriptive PR description - you can list all the tasks you have done in the project requirement.

**Review by bellom — APPROVED** [review]
@NurkaAmre 

### This is a manual review

Note: It will be approved on your dashboard as soon as possible. 

# Approved

Cheers and Happy coding!👏👏👏

**Pull request merged**